packages/tooltip/stories/tooltip.stories.ts

```diff
@@ -276,6 +276,7 @@ const overlaid = (openPlacement: Placement): TemplateResult => {
                 ['left', 'negative'],
                 ['right', 'positive'],
                 ['top', 'info'],
+                ['top-start', 'info'],
```

Comment:
We should add a story that uses this value by default so the `top-start` delivery is available in the VRTs.